ocaml/xapi-idl/rrd/rrd_interface.ml

```diff
@@ -181,12 +181,12 @@ module RPC_API (R : RPC) = struct
 
   let has_vm_rrd =
     let bool_p = Param.mk Types.bool in
-    declare "has_vm_rrd"
+    declare "has-vm-rrd"
```

Comment:
Hmm... does this actually work? I think these should match the functions in the OCaml interface.